mathml/presentation-markup/mpadded/mpadded-002.html

```diff
@@ -126,6 +133,7 @@
       <mpadded id="height0" height="25px"><mspace width="100px" depth="125px" height="150px" style="background: black; opacity: .5"/></mpadded>
       <mpadded id="height1" height="50px"><mspace width="100px" depth="125px" height="150px" style="background: black; opacity: .5"/></mpadded>
       <mpadded id="height2" height="75px"><mspace width="100px" depth="125px" height="150px" style="background: black; opacity: .5"/></mpadded>
+      <mpadded id="percentages" width="100%" height="100%" depth="100%"><mspace width="100px" depth="125px" height="150px" style="background: black; opacity: .5"/></mpadded>
```

Comment:
Maybe use different percentage values and something other than 100%? Otherwise, an mrow would just do the same, so it's not really testing attributes...

Reply:
Sure, I chose some different percentages now.